requirements/base.in

```diff
@@ -2,5 +2,6 @@
 
 celery >= 3.1,<5.0
 Django                    # Web application framework
+django-celery==3.2.1
```

Comment:
This and `freezegun` should be the only additions I made to requirements, all the other changes are the results of trying to unpack `pip-compile` files everywhere.